src/NHibernate/Util/StringHelper.cs

```diff
@@ -159,7 +159,7 @@ public static string ReplaceOnce(string template, string placeholder, string rep
 		}
 
 		/// <summary>
-		/// Just a fa�ade for calling string.Split()
+		/// Just a façade for calling string.Split()
```

Comment:
Each time I am looking into this file, it gets corrupted due to not being utf8 with that `ç`. Now re-encoded by the way.

Reply:
Facade, without the cedilla, is the most common (and correct) spelling anyway, so... ;)

Reply:
Now changed to `c`.